examples/website/scenegraph-layer/app.js

```diff
@@ -0,0 +1,151 @@
+/* global fetch, window */
+import React, {Component} from 'react';
+import {render} from 'react-dom';
+
+import {StaticMap} from 'react-map-gl';
+import DeckGL from '@deck.gl/react';
+import {ScenegraphLayer} from '@deck.gl/mesh-layers';
+
+import {GLBScenegraphLoader, GLTFScenegraphLoader} from '@luma.gl/addons';
+import {registerLoaders} from '@loaders.gl/core';
+
+registerLoaders([GLBScenegraphLoader, GLTFScenegraphLoader]);
+
+// Set your mapbox token here
+const MAPBOX_TOKEN = process.env.MapboxAccessToken; // eslint-disable-line
+// mapbox style file path
+const MAPBOX_STYLE =
+  'https://rivulet-zhang.github.io/dataRepo/mapbox/style/map-style-dark-v9-no-labels.json';
+
+const DATA_URL = 'https://opensky-network.org/api/states/all';
+
+export const INITIAL_VIEW_STATE = {
+  latitude: 39.1,
+  longitude: -94.57,
+  zoom: 3.8,
+  maxZoom: 16,
+  pitch: 0,
+  bearing: 0
+};
+
+const DATA_INDEX = {
+  ORIGIN_COUNTRY: 2,
+  LONGITUDE: 5,
+  LATITUDE: 6,
+  BARO_ALTITUDE: 7,
+  VELOCITY: 9,
+  TRUE_TRACK: 10,
+  GEO_ALTITUDE: 13,
+  POSITION_SOURCE: 16
+};
+
+export class App extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {};
+
+    if (!window.demoLauncherActive) {
+      this._loadData();
+    }
+  }
+
+  componentWillUnmount() {
+    if (this._animationFrame) {
```

Comment:
What is this for?

Reply:
Copied the whole file from another example. Forgot to delete this part.